ugvc/__main__.py

```diff
@@ -12,23 +12,31 @@
     sys.path.insert(0, path)
 
 from ugvc.cnv import filter_sample_cnvs
-
+from ugvc.joint import compress_gvcf
```

Comment:
this is the precommit's doing. 
I don't know how to prevent it from changing stuff